js/securesubmit/checkout-form.js

```diff
@@ -108,6 +108,7 @@ function securesubmitMultishipping(multiForm) {
     };
     return secureSubmit;
 }
+var amastyCompleteCheckoutOriginal;
```

Comment:
@ttnnkkrr Can we namespace this function name with a prefix of `securesubmit` or similar?